test/fixtures/export/Do not caps-protect name fields #384.noopsort.bibtex

```diff
@@ -119,42 +119,42 @@
 }
 
 @book{miesvanderrohe,
-  author = {Mies van der Rohe, Ludwig},
+  author = {{Mies van der Rohe}, Ludwig},
```

Comment:
multipart Lasts containing lowercase words (“Mies van der Rohe”) need to be wrapped in braces (unlike biblatex!)